src/blackmetal/src/communication/controlSW/Odometry.cpp

```diff
@@ -177,8 +191,11 @@ void Odometry::changeRobotLocation(Speed &&speed, long double &&elapsedTime)
 		SUCCESS("X: " << m_coordination.x);
 		m_coordination.y += dyt;
 		SUCCESS("Y: " << m_coordination.y);
-		m_coordination.angle += changeOfAngleInTime;
-		SUCCESS("Angle: " << m_coordination.angle);
+		m_coordination.z += changeOfAngleInTime;
+		SUCCESS("Angle: " << m_coordination.z);
+		if (!m_positionPublisher) {
```

Comment:
asi si chcel napisat. Teraz by ti to padlo
```suggestion
		if (m_positionPublisher) {
```